android/src/main/java/com/auth0/react/CredentialsParser.java

```diff
@@ -29,7 +29,8 @@ public static ReadableMap toMap(Credentials credentials) {
         map.putString(ID_TOKEN_KEY, credentials.getIdToken());
         map.putString(SCOPE, credentials.getScope());
         map.putString(REFRESH_TOKEN_KEY, credentials.getRefreshToken());
-        map.putString(TYPE_KEY, credentials.getType());
+        map.putString(TYPE_KEY, credentials.getType());//@deprecate This should be removed in next major (Reference - https://github.com/auth0/react-native-auth0/issues/614)
+        map.putString(TOKEN_TYPE_KEY, credentials.getType());
         return map;
```

Comment:
Should we be updating any tests?

Reply:
Since this is android specific and we don't have native unit tests, no unit tests need to be updated

Reply:
Since this is android specific and we don't have native unit tests, no unit tests need to be updated